include/swift/AST/Types.h

```diff
@@ -4260,7 +4309,7 @@ class SILParameterInfo {
   bool operator==(SILParameterInfo rhs) const {
     return getInterfaceType() == rhs.getInterfaceType() &&
            getConvention() == rhs.getConvention() &&
-           getDifferentiability() == rhs.getDifferentiability();
+           getOptions().toRaw() == rhs.getOptions().toRaw();
```

Comment:
I think in all the spots where `==` + `.toRaw()` it used you actually want to use `.containsOnly`.

Reply:
No. I am trying to compare against all possible flags.

Reply:
I misunderstood the API... you are correct.